src/App.vue

```diff
@@ -41,8 +41,11 @@ html {
   margin: 0;
 }
 
-#app {
+*:not(i){
```

Comment:
Maybe we can change default styles of element UI? Using * selector isn't good practice